assets/dev/scss/frontend/widgets/tabs.scss

```
@@ -3,6 +3,30 @@
 //
 
 .elementor-widget-tabs {
+	--tabs-wrapper-display: flex;
+	--tabs-wrapper-justify-content: start;
+	--tabs-wrapper-title-size: auto;
+	--tabs-border-width: 1px;
+	--tabs-border-color: #{$gray-light};
+	--tabs-background-color: transparent;
+
+	/* TODO Find better solution for all global defaults. */
+	--tabs-title-color: var( --e-global-color-primary );
+	--tabs-title-active-color: var( --e-global-color-accent );
+
+	--tabs-title-typography-font-family: var( --e-global-typography-primary-font-family );
+	--tabs-title-typography-font-size: initial;
+	--tabs-title-typography-font-weight: var( --e-global-typography-primary-font-weight );
```

```suggestion
	/* TODO Find better solution for all global defaults. */
	--tabs-title-color: var( --e-global-color-primary );
	--tabs-title-active-color: var( --e-global-color-accent );

	--tabs-title-typography-font-family: var( --e-global-typography-primary-font-family );
	--tabs-title-typography-font-size: initial;
	--tabs-title-typography-font-weight: var( --e-global-typography-primary-font-weight );
```

@matipojo need consult with you.